doc/lua-filters.md

```diff
@@ -399,7 +399,7 @@ colon syntax (`mystring:uc_upper()`).
 # Debugging Lua filters
 
 Many errors can be avoided by performing static analysis.
-[`luacheck`](https://github.com/rnwst/pandoc.git) may be used for this
+[`luacheck`](https://github.com/mpeterv/luacheck) may be used for this
```

Comment:
This URL is also incorrect. The old maintainer passed away several years ago (see the top issue in that repository) and nobody has access to it. This is the canonical repository now:

```suggestion
[`luacheck`](https://github.com/lunarmadules/luacheck) may be used for this
```